java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -49,6 +48,57 @@ protected ColumnView(long address) {
     this.nullCount = ColumnView.getNativeNullCount(viewHandle);
   }
 
+  /**
+   * Create a new column view based off of data already on the device. Ref count on the buffers
+   * is not incremented and none of the underlying buffers are owned by this view. If ownership
```

Comment:
I think the comment needs to emphasize that the resulting instance is only valid as long as the underlying buffers remain valid.  If someone closes those buffers while this view instance is still active and it is later accessed, the program as undefined behavior.